app/views/gene.html

```diff
@@ -1,466 +1,760 @@
 <div class="geneMainDiv" ng-if="!status.rendering">
```

Comment:
There seems to be some indentation formatting issue. Let's add a .editorconfig file to force the IDE follows the rule. Please see an example in oncokb repo.

Reply:
Copied from oncokb repo to keep code style consistency.